virttest/qemu_devices/qdevices.py

```diff
@@ -1844,7 +1844,7 @@ class QVirtioFSDev(QDaemonDev):
     Virtiofs pseudo device.
     """
 
-    def __init__(self, aobject, binary, sock_path, source, extra_options=None):
+    def __init__(self, aobject, binary, sock_path, source, extra_options=None, debug_mode=False):
```

Comment:
I have one question that how could I enable `debug_mode` in the case level?
From my side, I didn't find any interface to do that.
Thanks.

Reply:
@YongxueHong The following are my test results, of course, if there is a better method, I am happy to improve.

```
with_local_source:
    debug_mode = yes

```

/usr/libexec/virtiofsd --socket-path=/tmp/avocado__doqcd1k/avocado-vt-vm1-fs1-virtiofsd.sock -o source=/tmp/virtio_fs1_test -d -o cache=auto

Reply:
Thank you very much for your suggestions and support, update later.